packages/form-participation/test-web/MixinWithValidators.lit.test.ts

```diff
@@ -0,0 +1,169 @@
+import { elementUpdated, expect, fixture, fixtureCleanup } from '@open-wc/testing';
+import { FormControlMixin, Validator } from '../src';
+import { html, LitElement } from 'lit';
+import { customElement, query } from 'lit/decorators.js';
+import { spy } from 'sinon';
+
+const requiredValidator: Validator = {
+  attribute: 'required',
+  key: 'valueMissing',
+  message: 'This field is required',
+  callback(instance, value): boolean {
+    return !!value;
+  }
+};
+
+const otherValidator: Validator = {
+  attribute: 'other',
+  key: 'customError',
+  message: 'will not trigger',
+  callback() { return true; }
+}
+
+@customElement('form-control-lit-validators')
+class FormControlLitValidators extends FormControlMixin(LitElement) {
+  static get formControlValidators(): Validator[] {
+    return [requiredValidator, otherValidator];
+  }
+
+  @query('div')
+  validationTarget: HTMLElement;
+
+  render() {
+    return html`<div></div>`;
+  }
+
+  _onInput({ target }: Event & { target: HTMLInputElement }) {
+    this.value = target.value;
+  }
+}
+
+@customElement('with-checked')
+class FormControlLitCheckedValidators extends FormControlLitValidators {
+  checked = false;
+  value = 'foo';
+}
+
+describe('The FormControlMixin using LitElement', () => {
+  let form: Element;
+  let el: FormControlLitValidators;
+  let constructor: typeof FormControlLitValidators;
+
+  beforeEach(async () => {
+    form = await fixture(html`
+      <form>
+        <form-control-lit-validators
+          name="formControl"
+        ></form-control-lit-validators>
+      </form>
+    `);
+    // @ts-ignore
+    el = form.querySelector<FormControlLitValidators>('form-control-lit-validators');
```

Comment:
Does casting once here prevent needing to do it in multiple tests below?
```suggestion
    el = form.querySelector<FormControlLitValidators>('form-control-lit-validators') as FormControlLitValidators;
```